src/test/java/com/taskadapter/redmineapi/IssueManagerIT.java

```diff
@@ -303,6 +304,74 @@ public void testCreateIssueSummaryOnly() throws RedmineException {
         assertEquals(EXPECTED_AUTHOR_ID, issue.getAuthorId());
     }
 
+    @Test
+    public void testCreateIssueWithParam() throws RedmineException {
+        RequestParam param = new RequestParam("name", "value");
+        Issue issue = new Issue(transport, projectId)
+                .setSubject("This is the Issue with one param")
+                .create(param);
+        assertNotNull("Checking returned result", issue);
+        assertNotNull("New issue must have some ID", issue.getId());
+    }
+
+    @Test
+    public void testCreateIssueWithNullParams() throws RedmineException {
+        RequestParam param1 = new RequestParam("name1", "param1");
+        RequestParam param2 = null;
+        RequestParam param3 = new RequestParam("name3", "param3");
+        RequestParam param4 = new RequestParam("name4", "param4");
+        Issue issue = new Issue(transport, projectId).setSubject("This is the Issue with null params")
+                .create(param1, param2, param3, param4);
+        assertNotNull("Checking returned result", issue);
+        assertNotNull("New issue must have some ID", issue.getId());
+    }
+
+    @Test
+    public void testCreateIssueWithNullParams2() throws RedmineException {
+        RequestParam param1 = new RequestParam("name1", "param1");
+        RequestParam param2 = null;
+        RequestParam param3 = new RequestParam("name3", "param3");
+        RequestParam param4 = new RequestParam("name3", "param4");
+        Issue issue = new Issue(transport, projectId).setSubject("This is the Issue with null params")
+                .create(param1, param2, param3, param4);
+        assertNotNull("Checking returned result", issue);
+        assertNotNull("New issue must have some ID", issue.getId());
+    }
+
+    @Test
+    public void testCreateIssueWithNullParams3() throws RedmineException {
+        RequestParam param1 = new RequestParam("name1", "param1");
+        RequestParam param3 = new RequestParam("name3", "param3");
+        Issue issue = new Issue(transport, projectId).setSubject("This is the Issue with null params")
+                .create(param1, null, param3);
+        assertNotNull("Checking returned result", issue);
+        assertNotNull("New issue must have some ID", issue.getId());
+    }
```

Comment:
these positive tests have a lot of duplication in them.  it makes sense to extract the common parts into utility method(s)